change/@itwin-access-control-client-22d2f77d-9841-493b-a1b9-1fa1db0f5ebc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add custom message option when adding/inviting new users",
```

Comment:
Can we change this back to 

> Adding optional `customMessage` param to the `addITwinUserMembersAsync` function.

I think you can edit this directly, you don't have to rerun the `change` command